### PR TITLE
Profile bug fix

### PR DESCRIPTION
**Pull request by ishamoel**

### CODE REVIEW

File: net/ProfileViewController.m [hunk 1]
```diff
@@ -14,10 +14,14 @@
...
 #import "UIView+Addition.h"
 #import "UIViewController+backViewController.h"
+
 #import "IssueHistoryViewController.h"
 #import "NSString+stringIsEmpry.h"
 #import "User.h"
 #import "DescriptionViewController.h"
+#import "UIScrollView+getContentSize.h"
+#import "NetworkDataSorce.h"
+#import "DataSorceProtocol.h"
```
Comment (simple-user) [inline net/ProfileViewController.m]: DatasorceProtocol is already included in NetworkDataSorce ;)

File: net/ProfileViewController.m [hunk 2]
```diff
@@ -297,27 +258,27 @@ - (void) sendProfileImage: (UIImage *)profileImage {
                           //Update the progress view
                           [weakSelf.progressView setProgress:uploadProgress.fractionCompleted animated:YES];
                           if (uploadProgress.fractionCompleted == 1.0) {
-                              // do something
+
```
Comment (simple-user) [inline net/ProfileViewController.m]: Why do you need condition witch doesn't do anything? ;)

File: net/ProfileViewController.m [hunk 3]
```diff
@@ -426,4 +387,62 @@ - (IBAction)changeUserDatails:(UIButton *)sender {
...
+- (void)textFieldDidEndEditing:(UITextField *)textField
+{
+    self.activeField = nil;
+}
+
+- (IBAction)dismissKeyboard:(id)sender
+{
+    [self.activeField resignFirstResponder];
+}
```
Comment (simple-user) [inline net/ProfileViewController.m]: it is have sense to set activeField to nil. May be its not necessarily but imho will be good )

Comment (simple-user) [inline net/ProfileViewController.m]: in dismissKeyboard

File: net/ProfileViewController.m (hunk 2 above)
Comment (ishamoel) [inline net/ProfileViewController.m]: I forgot to set progress

File: net/ProfileViewController.m (hunk 1 above)
Comment (ishamoel) [inline net/ProfileViewController.m]: Yep, i already remove it

**Pull request merged**